pgcli/main.py

```diff
@@ -793,11 +795,13 @@ def get_last_query(self):
 @click.option('--prompt', help='Prompt format (Default: "\\u@\\h:\\d> ").')
 @click.option('-l', '--list', 'list_databases', is_flag=True, help='list '
               'available databases, then exit.')
+@click.option('--auto-vertical-output', is_flag=True,
+              help='Automatically switch to vertical output mode if the result is wider than the terminal width.')
```

Comment:
This should probably say "expanded mode" and not "vertical output mode". Just a minor difference between mysql and psql.